Makefile

```
@@ -22,5 +22,5 @@ deploy: build deploy-setup
 	rm .gcloudignore
 	rm main.py
 	rm requirements.txt
-format:
-	black . -l 79
+lint:
```

i'd suggest we keep this as make format given your change to ensure it formats code as well as identifying lint issues (fwiw i [asked chatgpt](https://chat.openai.com/share/25354619-d9b4-4b1d-8e8b-40a69d4c1c7f) to check my intuition here)

---

Got it. Just added 'black' back while keeping the `npm run lint`

---

Sorry I meant to label `make format` if we're writing (not just checking). May as well keep as one command with black too:
```suggestion
    npm run lint
```